src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,102 @@
 import React from 'react';
+import ProductList from './ProductList';
+import { useEffect, useState } from 'react';
 import './Cart.scss';
 
 const Cart = () => {
-  return <div>장바구니</div>;
+  const [orderList, setOrderList] = useState([]);
+  const [address, setAddress] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/cart.json')
+      .then(res => res.json())
+      .then(data => {
+        setOrderList(data);
+      });
+  }, []);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/signup.json')
+      .then(res => res.json())
+      .then(data => {
+        setAddress(data);
+      });
+  }, []);
+
+  const orderConfirm = () => {
+    if (window.confirm('주문하시겠습니까?')) {
+      alert('주문이 완료 되었습니다');
+    } else {
+      alert('주문이 취소 되었습니다');
+    }
+  };
+
+  const deleteConfirm = () => {
+    if (window.confirm('삭제 하시겠습니까??')) {
+      // let copy = [...orderList];
+      // copy.splice(0, 1);
+      // setOrderList(copy);
+    }
+  };
+
+  let count = 0;
+
+  return (
+    <div className="cart">
+      <div>nav</div>
+      <div className="cartMain">
+        <div className="cartName">장바구니</div>
+        <div className="allCheckBox">
+          <div className="label">
+            <input type="checkbox" className="woo" />
+            <p>전체선택</p>
+          </div>
+          <button className="selectDeleteBtn" onClick={deleteConfirm}>
+            선택삭제
+          </button>
+        </div>
+        <div>
+          <p className="cartInProducts">
+            <strong>장바구니에 담긴 상품</strong>
+            <button>V</button>
+          </p>
+        </div>
+        {orderList.map((order, i) => {
+          count = count + order.price * order.quantity;
+          return (
+            <ProductList
+              key={order.id}
+              order={order}
+              deleteConfirm={deleteConfirm}
```

Comment:
리뷰대로 구현하신다면 이 부분에서 바로 id를 보내주시면 됩니다.
```suggestion
              deleteConfirm={() => deleteConfirm(order.id)}
```